### PR TITLE
Add Hidden Icons extension

### PR DESCRIPTION
**Pull request by koinzhang**
## Description

A casual game, find out the hidden icons in 30s.

**Commands**:

- **Find out Hidden Icon**: Game entrance, use this command to enter the game
  - Start Game: Start the game, the game difficulty is divided into four levels
  - History Score：View history high scores

## Screencast

Screenshot1

Screenshot2

Screenshot3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/hidden-icons/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "icon": "find-out-icon.png",
+  "author": "koinzhang",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "find-out-hidden-icon",
+      "title": "Find out Hidden Icon",
```
Comment (fe9lix) [inline extensions/hidden-icons/package.json:15]:
```suggestion
      "title": "Find Hidden Icon",
```

File: extensions/hidden-icons/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hidden-icons",
+  "title": "Hidden Icons",
+  "description": "A casual game, find out the hidden icons in 30s.",
```
Comment (fe9lix) [inline extensions/hidden-icons/package.json:5]:
```suggestion
  "description": "A casual game where you need to find a hidden icon.",
```

File: extensions/hidden-icons/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "find-out-hidden-icon",
+      "title": "Find out Hidden Icon",
+      "subtitle": "Hidden Icons",
+      "description": "A casual game, find out the hidden icons in 30s.",
```
Comment (fe9lix) [inline extensions/hidden-icons/package.json:17]:
```suggestion
      "description": "A casual game where you need to find a hidden icon.",
```

File: extensions/hidden-icons/src/find-out-hidden-icon.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+            <ActionPanel>
+              <Action
+                title={"Start Game"}
+                icon={Icon.Binoculars}
+                onAction={() => {
+                  setIsGaming(true);
+                }}
+              />
+              <Action
+                title={"History Score"}
```
Comment (fe9lix) [inline extensions/hidden-icons/src/find-out-hidden-icon.tsx:108]:
```suggestion
                title={"Score History"}
```

File: extensions/hidden-icons/src/find-out-hidden-icon.tsx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+      ) : (
+        findOutIcons.map((value, index) => (
+          <List.Item
+            key={"index" + index}
+            icon={" "}
+            title={value}
+            actions={
+              <ActionPanel>
+                <Action
+                  title={isGaming ? "Find out " + targetIcon + ". Time left: " + leftTime + "s" : "Start Game"}
```
Comment (fe9lix) [inline extensions/hidden-icons/src/find-out-hidden-icon.tsx:128]:
```suggestion
                  title={isGaming ? "Find " + targetIcon + ". Time left: " + leftTime + "s" : "Start Game"}
```

File: extensions/hidden-icons/src/score-page.tsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  return (
+    <Detail
+      markdown={`# **${breakRecord ? "🎉 " : ""}Your Score${breakRecord ? " 🎉" : ""}**  
+${mode} Mode
+
+${scoreCanvas} 
+`}
+      metadata={
+        <Detail.Metadata>
+          <Detail.Metadata.Label title="Highest score in history" text={`---------------------`} />
```
Comment (fe9lix) [inline extensions/hidden-icons/src/score-page.tsx:40]:
```suggestion
          <Detail.Metadata.Label title="Highest Score in History" text={`---------------------`} />
```

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @koinzhang, sorry for the late review! I've left a couple of wording suggestions. Also, how about changing the Toast style to a regular success Toast that just appears when the user scores? (Animated Toasts are meant to be used more as short/temporary progress indicators...)

**Comment by koinzhang** [comment]
Hi @fe9lix. Thanks for your review.
Thank you very much for the suggestion, I have submitted the commit for update.

> Also, how about changing the Toast style to a regular success Toast that just appears when the user scores? (Animated Toasts are meant to be used more as short/temporary progress indicators...)

Update: The live scores are now displayed in the action title no longer using Toast.

Please continue to review.

**Pull request merged**